.github/workflows/docker-merge-tags.yml

```diff
@@ -18,7 +18,7 @@ on:
       timeout-minutes:
         description: Timeout in minutes
         type: number
-        default: 25
```

Comment:
These timeouts were too big and almost always it should end in ~6 minutes.
With retry 10 should be enough.